hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -843,14 +853,39 @@ public String getCheckCondition(String columnName, long min, long max) {
 	 * @return a SQL expression that will occur in a {@code check} constraint
 	 */
 	public String getCheckCondition(String columnName, long[] values) {
```

Comment:
Please deprecate this method for removal and point to the new method.

Reply:
done